kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -102,7 +102,7 @@ import * as contentNode from '../utils/contentNodeUtils';
 import InteractionList from '../views/InteractionList';
 import ExamReport from '../views/ExamReport';
 import SlotTruncator from '../views/SlotTruncator';
-import TextTruncatorCss from '../views/TextTruncatorCss';
+import KTextTruncator from 'kolibri-design-system/lib/KTextTruncator';
```

Comment:
I believe `K` components should work without being defined in `apiSpec.js` since they're all available from kolibri-design-system for all Kolibri files. Can you try to remove and check if it still works as expected?

Reply:
Thanks for the feedback, I've updated the PR to remove all these imports.

Reply:
Thank you for the feedback! Removing it from `apiSpec.js` works as expected.